packages/aws-cdk-lib/aws-ec2/README.md

```diff
@@ -2472,9 +2472,15 @@ Create and manage customer-managed prefix lists. If you don't specify anything i
 You can also create an empty Prefix List with only the maximum number of entries specified, as shown in the following code. If nothing is specified, maxEntries=1.
 
 ```ts
-new ec2.PrefixList(this, 'EmptyPrefixList', {
+declare const sg: ec2.SecurityGroup;
+
+const prefixList = new ec2.PrefixList(this, 'EmptyPrefixList', {
   maxEntries: 100,
 });
+
+// Attach to Security Group
+sg.addIngressRule(prefixList.peer, Port.tcp(80));
+sg.connections.allowFrom(prefixList.peer, Port.tcp(443));
```

Comment:
Can we make PrefixList class directly implement `IPeer`? Just like security groups:
https://github.com/aws/aws-cdk/blob/6303b72580e0381dcbb291ad7f424cfd4b5de329/packages/aws-cdk-lib/aws-ec2/lib/security-group.ts#L18

Then we can write it like: `sg.connections.allowFrom(prefixList, Port.tcp(443));`